### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Ntwali-Josue**

### CODE REVIEW

File: src/components/ProductItem.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            src={deleteIcon}
+            alt="delete icon"
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+export default ProductItem;
```
Comment (Emmyn5600) [inline src/components/ProductItem.js:72]: - [ ] When you are moving a cursor the selected/pointed card before has to show all effect that is happening on each card like delete icon, hover color, and black color before the title.

test11

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @Ntwali-Josue 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet, please work on it and ask for another review.


 ## Required Changes ♻️
_Check the comments under the review._

File: src/components/ProductsContainer.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+          products={this.state.products}
+          deleteProductProps={this.deleteProduct}
+          updateProductName={this.updateProductName}
+          updateProductDescription={this.updateProductDescription}
+          updateProductID={this.updateProductID}
+        />
+      </div>
+    );
+  }
+}
```
[the remark this answers is not in the document]
Comment (Ntwali-Josue) [reply src/components/ProductsContainer.js:143]: @Emmyn5600 It seems that I do not have the same design you are showing to me if you can see here I only have inputs to use for adding outcomes, and also three dots that do not have anything on it. You can check all the slides [here](https://xd.adobe.com/view/2c5ab9e1-8364-41d1-94f1-3c2cba482893-7909/specs/)

image

File: src/components/ProductsList.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+              updateProductDescription={this.props.updateProductDescription}
+              updateProductID={this.props.updateProductID}
+            />
+          ))}
+        </ul>
+      </div>
+    );
+  }
+}
+
```
[the remark this answers is not in the document]
Comment (Ntwali-Josue) [reply src/components/ProductsList.js:24]: Sure bro.

**Pull request merged**